docker/Dockerfile.rocm

```diff
@@ -58,6 +58,11 @@ RUN git clone ${ATER_REPO} \
     && git submodule update --init --recursive \
     && PREBUILD_KERNELS=1 GPU_ARCHS=gfx942 python3 setup.py develop
 
+# Copy config files to support MI300X in virtualized environments (MI300X_VF).  Symlinks will not be created in image build.
+RUN find /sgl-workspace/sglang/python/sglang/srt/layers/quantization/configs/ \
+         /sgl-workspace/sglang/python/sglang/srt/layers/moe/fused_moe_triton/configs/ \
+         -type f -name '*MI300X*' | xargs -I {} sh -c 'vf_config=$(echo "$1" | sed "s/MI300X/MI300X_VF/"); cp "$1" "$vf_config"' -- {}
```

Comment:
Tested this out on 8*MI300X machine in Azure, resulted in the configs actually being picked up and jumped single sequence gen throughput from 17tok/s to 28tok/s!